website/docs/sql/datatype/index.md

```diff
@@ -14,6 +14,7 @@ Name|Aliases|Description
 `CHARACTER [ (n) ]`|`CHAR [ (n) ]`|fixed-length character string
 `CHARACTER VARYING (n)`|`VARCHAR (n)`|variable-length character string with limit
 `DATE`||calendar date (year, month, day)
+`REAL`|`FLOAT4`|single precision floating-point number (4 bytes)
 `DOUBLE PRECISION`|`FLOAT8`|double precision floating-point number (8 bytes)
```

Comment:
Do we want to mention `FLOAT` here?

Reply:
86d12d7ad9f81cd1d90835797b9a8a79fd93f48d